packages/apps/contracts/CommitRevealApp.sol

```diff
@@ -58,7 +58,7 @@ contract CommitRevealApp {
     return Player.CHOOSING;
   }
 
-  function reducer(AppState state, Action action)
+  function reduce(AppState state, Action action)
```

Comment:
Is there a reason we don't just call this `applyAction` or something? Is it just to use Redux terminology?

Reply:
I prefer `applyAction` over `reduce`

Reply:
@snario @armaniferrante @ebryn

Reply:
As mentioned offline, I prefer `applyAction` over reduce as well. Reduce/reducer confused me when I first read it.